lib/plugin/version/Version.js

```diff
@@ -111,6 +110,10 @@ class Version extends Plugin {
       return semver.inc(latestVersion, 'prerelease', preReleaseId);
     }
 
+    if (isCI && !increment) {
```

Comment:
Thanks for your PR, @pstrh! I see the issue now and like the solution. If you could please use `this.config.isCi` here directly instead of passing the argument around (and ideally also run `npm run format`) I'm happy to merge it.

(Looks like we can even remove `getIncrementedVersionCI` from this plugin then, right?)